packages/core/src/index.ts

```diff
@@ -15,4 +15,6 @@ export * from "./types/module-builder";
 export * from "./types/provider";
 export * from "./types/serialization";
 export * from "./types/status";
+export * from "./types/verify";
+export { verify } from "./verify";
```

Comment:
I found this name a bit confusing because you are the one who is doing the verification, and this verb seems to indicate that calling it will run the verification. Let's rename it to something like `getVerificationInformation`.